create-doi-request/src/test/java/no/unit/nva/doi/requests/handlers/CreateDoiRequestHandlerTest.java

```diff
@@ -67,7 +67,7 @@ public CreateDoiRequestHandlerTest() {
     public void init() {
         initializeDatabase();
         Clock mockClock = Clock.fixed(mockNow, ZoneId.systemDefault());
-        doiRequestsService = new DynamoDBDoiRequestsService(client, objectMapper, environment, mockClock);
+        doiRequestsService = new DynamoDBDoiRequestsService(client, environment, mockClock);
```

Comment:
objectMapper is always the JsonUtils.objectMapper except for one test. We use reflection in the test.